core/web/service_agreements_controller_test.go

```diff
@@ -28,7 +28,7 @@ func TestServiceAgreementsController_Create(t *testing.T) {
 	require.NoError(t, app.Start())
 
 	client := app.NewHTTPClient()
-	base := string(cltest.MustReadFile(t, "testdata/hello_world_agreement.json"))
```

Comment:
Can we remove the `testdata/hello_world_agreement.json` fixtures for this directory now?

Reply:
Nope, it's still used by other tests.